.github/workflows/workflow.yaml

```diff
@@ -113,3 +113,46 @@ jobs:
         run: make test.e2e
         env:
           CGO_ENABLED: "0"
+
+  load-tests:
+    strategy:
+      fail-fast: false
+      matrix:
+        qps: [1,10,100]
```

Comment:
Do you really think that 1 QPS plus 256 payload is meaningful while we have 100 QPS plus 8192?